bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/DiscoveryResource.java

```diff
@@ -88,7 +89,23 @@ public void onFinished() {
                 logger.debug("Scan for binding '{}' successfully finished.", bindingId);
             }
         });
-        return Response.ok().build();
+        
+        return Response.ok(new ScanResponseResultDto(discoveryServiceRegistry.getMaxScanTimeout(bindingId))).build();
+    }
+    
+    /**
+     * Dummy class needed to create a proper JSON formating of the response
+     * 
+     * @author ivaylo.ivanov
+     *
+     */
+    @SuppressWarnings("unused")
+    private static final class ScanResponseResultDto {
```

Comment:
Then you did something wrong. It is of course possible to return PLAIN values as an HTTP response, we are e.g. doing it [here](https://github.com/eclipse/smarthome/blob/master/bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/item/ItemResource.java#L205).